client/src/components/Events/Event.js

```diff
@@ -1,25 +1,30 @@
-import React from "react";
-import { Link } from "react-router-dom";
+import React from 'react';
+import { Link } from 'react-router-dom';
+import '../../styles/events.css';
 
 // Event appears on EventListPage
 const Event = ({ id, date, time, location, eventTitle }) => {
-	const eventStyle = {
-		border: '1px solid black',
-		padding: '15px',
-		marginTop: '8px',
-		marginLeft: '4px',
-		marginRight: '4px',
-	}
+  //const eventStyle = {
+  //border: '1px solid black',
+  //padding: '15px',
+  //marginTop: '8px',
+  //marginLeft: '4px',
+  //marginRight: '4px',
+  //}
```

Comment:
If code is unused, delete it. We have Git history if we need to retrieve it again.